packages/haste-task-read/README.md

```diff
@@ -0,0 +1,22 @@
+# haste-task-read
+Haste task for reading files from the file system.
+
+The task returns a promise with an array of virtual files:
+
+* A virtual file consists of two things:
+  1. filename `<string>` - an absolute path to the file.
+  2. content `<string>` - the content of the file encoded to `utf-8`.
+```js
+run(read({ pattern: '**.*'}))
+  .then(virtualFiles => console.log(virtualFiles))
+
+// [{ filename: "/path/to/file.js", content: "file content"}]
+
+```
+## options
+
+#### pattern
+
+Type: `string`
+
+A glob pattern, uses [globby](https://github.com/sindresorhus/globby) under the hood.
```

Comment:
`globby` is just a wrapper around [node-glob](https://github.com/isaacs/node-glob). The wrapper may change, I think it would be better to direct readers to node-glob (which also includes an explanation on the supported glob patterns).

Reply:
Yes, globby uses node-glob, but it uses some other libraries as well.
It provides some additional features, like the one that you've mentioned, "the pattern can be an array". 

This task is a wrapper on globby's main functionality, so it's clearer to give it as a reference.

btw, globby has some [nice explanations](https://github.com/sindresorhus/globby#globbing-patterns) as well